### ISSUE TITLE
docs for Caddy cheatsheet

### ISSUE THREAD

**Issue by coolaj86** [webinstall/webi-installers#390]
Needs how to run a quick-n-dirty local server:

```txt
# local-ish http-only file browser
# caddy hash-password
http://example.duckdns.org:3000 {
	basicauth / {
		myusername JDJhJDE0JFVMRGJ3TVR4YXlOc2x5U2N6cldPSE9kOVFaZzZ0V2JjSThySEFVbGQuTWljT0pCYVBzamk2
	}
	basicauth /* {
		myusername JDJhJDE0JFVMRGJ3TVR4YXlOc2x5U2N6cldPSE9kOVFaZzZ0V2JjSThySEFVbGQuTWljT0pCYVBzamk2
	}
	root * /Volumes/Storage/Shared/
	file_server {
	  browse
    }
}
```

**Comment by coolaj86** [comment]
# Exclude Dev Sites from Search Engines

```caddy
dev.example.com {
    header {
        Link "<https://production.example.com{http.request.orig_uri}>; rel=\"canonical\""
        X-Robots-Tag noindex
    }
}
```

**Comment by coolaj86** [comment]
Included in #649
